src/applications/personalization/view-dependents/containers/ViewDependentsApp.jsx

```diff
@@ -30,13 +30,11 @@ class ViewDependentsApp extends Component {
           {
             name: 'Billy Blank',
             social: '312-243-5634',
```

Comment:
Just curious, did we ever confirm what the data for view dependents will look like?

Reply:
I am working on a separate branch to get real data now.